pyvista/plotting/mapper.py

```diff
@@ -29,6 +29,20 @@ def __init__(self, theme=None, **kwargs):
         self._theme = theme
         self.lookup_table = LookupTable()
 
+    @property
+    def bounds(self) -> tuple:
```

Comment:
This should return `Tuple[Number, Number, Number, Number, Number, Number]` or this needs to be fixed in #3777